ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiTimeService.scala

```diff
@@ -139,6 +139,7 @@ private[apiserver] final class ApiTimeService private (
   override def close(): Unit = {
     super.close()
     dispatcher.close()
+    ()
```

Comment:
Why not wait for the `dispatcher.close()` ?

Reply:
`AutoCloseable.close` doesn't say anything about blocking, but it looks safer to me to wait. For other places where we block see for example `AkkaExecutionSequencerPool`, `QueueBackedTracker`, or `BuffersUpdater`.